src/modules/launcher/Wox.Plugin/Wox.Plugin.csproj

```diff
@@ -49,6 +49,15 @@
     <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
   </PropertyGroup>
 
+  <PropertyGroup>
+    <DefaultItemExcludes>$(DefaultItemExcludes);**/*.Template.xaml</DefaultItemExcludes>
+  </PropertyGroup>
+  <ItemGroup>
+    <EmbeddedResource Include="**/*.Template.xaml" />
```

Comment:
Just CYA’ing. Are these files on disk and needed by the app?  They are not in the WSX

Reply:
@crutkas These are embedded files which are used by the ControlzEx ThemeManager, no need to add this to the wsx.